Formula/ghostunnel.rb

```diff
@@ -0,0 +1,20 @@
+class Ghostunnel < Formula
+  desc "Simple SSL/TLS proxy with mutual authentication"
+  homepage "https://github.com/ghostunnel/ghostunnel"
+  url "https://github.com/ghostunnel/ghostunnel/archive/refs/tags/v1.5.3.tar.gz"
+  sha256 "008b96d507b80480f4e78d31bdfbb37b794f51c2a7a80a8f3561844e2739b5a5"
+  license "Apache-2.0"
+  head "https://github.com/ghostunnel/ghostunnel.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "ghostunnel"
+    bin.install "ghostunnel"
+  end
+
+  test do
+    output = shell_output("#{bin}/ghostunnel --help 2>&1")
+    assert_match "client", output
```

Comment:
Would it be possible to have a test that ensures it fails as expected because, e.g., there isn't a server running?

Reply:
The issue is ghostunnel won't initiate a server connection until a client connects to it.  Because it blocks, and cannot be piped to, getting a client to connect would require backgrounding ghostunnel and spawning `nc` separately.  How do you safely background processes in a Homebrew `test`, ensuring they're always killed/destroyed?

Reply:
Alright I cooked something up.